### ISSUE TITLE
Update regex to 2014.12.24

### ISSUE THREAD

**Issue by mplewis** [mailgun/talon#30]
Hello!

I tried installing talon in Python 3 today and ran into the following issue installing regex:

```
Downloading/unpacking regex==0.1.20110315
  Downloading regex-0.1.20110315.tar.gz (948kB): 948kB downloaded
  Running setup.py (path: .../regex/setup.py) egg_info for package regex
    No unicodedata_db.h could be prepared for Python 3.4.2 (default, Oct 19 2014, 17:55:38)
    [GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.54)]
    Complete output from command python setup.py egg_info:
    No unicodedata_db.h could be prepared for Python 3.4.2 (default, Oct 19 2014, 17:55:38)

[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.54)]

----------------------------------------
Cleaning up...
```

I don't have this problem installing regex==2014.12.24. Could you please consider updating the dependency version?

I would be happy to bump the version, ensure tests pass, and open a PR. Thanks!

**Comment by afedosenko** [comment]
+1

**Comment by JordanReiter** [comment]
:+1:

**Comment by tgwizard** [comment]
Hi,

I've created a fork of talon, https://github.com/tictail/claw, in which I've removed the signature extraction that depends on ML (which I didn't use), and cleaned up some dependencies (now much less weird stuff installed). I've also started to implement some bug fixes, along with CI test suites and so on.

I've updated the regex dependency to `regex>=2015.07.19`. The only thing standing in the way for proper 3 support is that mailgun/flanker is used in tests, and flanker is 2.7 only.
